sonic-psud/scripts/psud

```diff
@@ -465,16 +465,19 @@ class DaemonPsud(daemon_base.DaemonBase):
             power = try_get(psu.get_power, NOT_AVAILABLE)
 
         if index not in self.psu_status_dict:
-            self.psu_status_dict[index] = PsuStatus(self, psu)
+            self.psu_status_dict[index] = PsuStatus(self, psu, index)
 
         psu_status = self.psu_status_dict[index]
         set_led = self.first_run
-        if psu_status.set_presence(presence):
+        presence_changed = psu_status.set_presence(presence)
```

Comment:
I can not reproduce that issue because that issue is only on DELL platform. However, please refer to line 480, I suppose it covers the issue you mentioned.

Reply:
with the presence of first_run, psu fan data is written in STATE_DB. Hence this change should be okay to merge.